src/plone/meta/re_enable_actions.py

```diff
@@ -0,0 +1,63 @@
+from .shared.call import call
+from .shared.packages import list_packages
+import argparse
+import itertools
+import pathlib
+
+
+org = 'plone'
+base_url = f'https://github.com/{org}'
+base_path = pathlib.Path(__file__).parent
+types = ['default']
+
+
+def run_workflow(base_url, org, repo):
+    """Manually start the tests.yml workflow of a repository."""
+    result = call('gh', 'workflow', 'run', 'tests.yml', '-R', f'{org}/{repo}')
+    if result.returncode != 0:
+        print('To enable manually starting workflows clone the repository'
+              ' and run config-package on it.')
```

Comment:
How do you do that? Should we refer to the README, so we don't have to maintain this help?

Also it's been renamed.

```suggestion
              ' and run config_package on it.')
```

Reply:
It has not been renamed.  Well, the file has been renamed.  Ah, that would be because `config-package.py` is a fine name for a script, but it is not valid as a module name: it cannot be imported. 

But in `setup.py` the PR now registers this command:

    config-package=plone.meta.config_package:main

So this creates a `./venv/bin/config-package` script.

I have pushed a change so here in the `re_enable_actions` module we take the path of the current command (expected to be something like `./venv/bin/re-enable-actions`) and replace `re-enable-actions` with `config-package`.